docs/src/components/RightSidebar/ThemeToggleButton.tsx

```diff
@@ -30,10 +30,10 @@ const icons = [
   </svg>,
 ];
 
-const ThemeToggle: FunctionalComponent = () => {
+function ThemeToggle() {
   const [theme, setTheme] = useState(() => {
     if (import.meta.env.SSR) {
-      return undefined;
+      return 'light';
```

Comment:
Returning `undefined` just loaded the page without either selected. Seemed like weird behavior.

Reply:
I think this is correct behavior, it's meant to respect your system preference by default.

Reply:
Oh. Hm. How would the system preference highlight the icon below? We have that `window.matchMedia` check below which should handle that?

Reply:
I’ll revert, then. Maybe neither icon being highlighted is OK until it’s hydrated

Reply:
We used to have an icon for system preference. It got removed along the way. It should exist in some form, but can be a follow-up.

Reply:
WFM. I’ll leave this as-is then and we can address later.